hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/client/rpc/TestECKeyOutputStream.java

```diff
@@ -229,6 +237,40 @@ public void testMultipleChunksInSingleWriteOp(int numChunks)
     validateContent(inputData, bucket, bucket.getKey(keyName));
   }
 
+  @Test
```

Comment:
I have a question on this - is the container key count only incremented when the block is closed via sending the close flag? Is that why this test is checking the key count increases after writing a key?

Reply:
Yes, you are right.